libs/asset_type_ec2.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+"""
+Asset types EC2 class
+
+Copyright 2020-2021 Leboncoin
+Licensed under the Apache License, Version 2.0
+Written by Nicolas BEGUIER (nicolas.beguier@adevinta.com)
+"""
+
+from .asset_type import AssetType
+
+# Debug
+# from pdb import set_trace as st
+
+class EC2(AssetType):
+    """
+    EC2 Asset Type
+    """
+    def __init__(self, name: str, private_ip: str, public: bool=False):
+        AssetType.__init__(self, name, public=public)
```

Comment:
May be changed to use `super().__init__(name, public=public)` as it is intended for  this :)

Reply:
Thanks !